mcs/db_cluster.go

```diff
@@ -26,7 +26,7 @@ func flattenDatabaseClusterShard(inst dbClusterInstanceResp) map[string]interfac
 func getClusterStatus(c *dbClusterResp) string {
 	instancesStatus := string(dbInstanceStatusActive)
 	for _, inst := range c.Instances {
-		if inst.Status == "error" {
+		if inst.Status == "ERROR" {
```

Comment:
maybe for more consistency, add the **dbInstanceStatusError** constant and use it